services/provider/server/server.go

```diff
@@ -937,13 +939,15 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					}),
 				},
 				&pb.ExternalResource{
-					Name: "cephfs",
+					Name: fmt.Sprintf("%s-groupsnapclass", req.StorageClaimName),
 					Kind: "VolumeGroupSnapshotClass",
 					Labels: map[string]string{
 						ramenDRStorageIDLabelKey: storageID,
 					},
 					Data: mustMarshal(map[string]string{
 						"csi.storage.k8s.io/group-snapshotter-secret-name": provisionerSecretName,
+						"fsName":    subVolumeGroup.Spec.FilesystemName,
+						"clusterID": clientProfile,
```

Comment:
do we set clusterID as clientProfile for other resouces as well?

Reply:
We have started setting it for VolumeReplicationClass with RDR since with convergence the clientProfile name will be sent from the provider.

Reply:
Resolving since Madhu cannot resolve the comments